features/step_definitions/events.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const request = require('../support/request');
+
+module.exports = function server() {
+    // eslint-disable-next-line new-cap
+    this.Before({
+        tags: ['@events']
+    }, () => {
+        this.repoOrg = 'screwdriver-cd-test';
+        this.repoName = 'functional-events';
+        // Reset shared information
+        this.pipelineId = null;
+        this.eventId = null;
+    });
+
+    this.Given(/^an existing pipeline with the workflow:$/,
+        { timeout: 60 * 1000 }, table =>
```

Comment:
Make `{ timeout: 60 * 1000 }` a constant since you redefine it for nearly every block